tensorflow/compiler/xla/service/gpu/gpu_layout_assignment.cc

```diff
@@ -71,6 +71,10 @@ HeuristicLayoutAssignment(const HloInstruction* instr,
     return kAllNCHW;
   }
 
+  if (instr->operand(0)->shape().element_type() == xla::PrimitiveType::CUS) {
+    return kAllNHWC;
```

Comment:
is this because cutlass always takes input in NHWC format?

Reply:
Yes.